File: Modul-1/Soal-8/PRAK108-2410817210027-HafizPerdana.py
import math

putaran = 5
jarak_tempuh = 14
phi = math.pi

keliling_satu_putaran = jarak_tempuh / putaran

jari_jari = keliling_satu_putaran / (2 * phi)

print("Diketahui :")
print("Pak Dengklek mengelilingi taman =", putaran, "Putaran")
print("Jarak tempuh Pak Dengklek =", jarak_tempuh, "Kilometer")
print("\nJawaban :")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jari_jari:.2f}", "Kilometer")

File: Modul-1/Soal-8/PRAK108-2410817210027-HafizPerdana.c
#include <stdio.h>
#include <math.h>

int main() {
    int putaran = 5;
    float jarak_tempuh = 14;
    const float phi = 3.14;
    float keliling_satu_putaran, jari_jari;

    keliling_satu_putaran = jarak_tempuh / putaran;
    jari_jari = keliling_satu_putaran / (2 * phi);

    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %d Putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.f Kilometer\n", jarak_tempuh);
    printf("\nJawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer", jari_jari);

    return 0;
}